### PR TITLE
Form->JSON->XML Mapping Fixes

### PR DESCRIPTION
**Pull request by markolson**
- Link disabledInLineOfDuty to militaryServiceInfo/dischargeDueToDisability
- Set discloseFinancialInformation based on the presence of any financial data

### CODE REVIEW

File: src/server/enrollment-system.js [hunk 1]
```diff
@@ -829,25 +841,33 @@ function veteranToEnrollmentDeterminationInfo(veteran) {
 //  * financialStatementInfo / spouseFinancialsCollection / spouseFinancialsInfo / incomeCollection / incomeInfo / type, Not applicable, Yes, Data element is not a form captured element but provides the income type to identify the value as the Spouse's gross income from employment.
 //  * financialStatementInfo / spouseFinancialsCollection / spouseFinancialsInfo / incomeCollection / incomeInfo / type, Not applicable, Yes, "Data element is not a form captured element but provides the income type to identify the value as the Spouse's gross income from FARM,  RANCH,  PROPERTY OR BUSINESS."
 function veteranToFinancialsInfo(veteran) {
+  const _expenses = resourceToExpenseCollection({
```
Comment (robbiethegeek) [inline src/server/enrollment-system.js]: is the reason to use the _expenses to not trigger the ESLint rule for expenses: expenses ?

Comment (markolson) [inline src/server/enrollment-system.js]: Yeah - is there a more idiomatic way to do that, besides quoting the keyname?

Comment (robbiethegeek) [inline src/server/enrollment-system.js]: const name = 'robbie';

const Robbie = { name }

Robbie.name
"robbie"

**Comment by robbiethegeek** [comment]
LGTM, will merge if all tests pass!

**Pull request merged**